src/api/app/views/webui/users/notifications/_notification_filter.html.haml

```diff
@@ -4,99 +4,99 @@
                                                             key: 'kind[build_failures]', name: 'kind[]', value: 'build_failures',
                                                             label_icon: 'fas fa-xmark text-danger',
-                                                            checked: @selected_filter[:kind]&.include?('build_failures') }
-    - if ReportPolicy.new(@user, Report).notify?
+                                                            checked: selected_filter[:kind]&.include?('build_failures') }
+    - if ReportPolicy.new(user, Report).notify?
       .px-4.py-1
-        = render partial: 'webui/shared/check_box', locals: { label: 'Reports', amount: @count['reports'],
+        = render partial: 'webui/shared/check_box', locals: { label: 'Reports', amount: counted_notifications['reports'],
                                                               key: 'kind[reports]', name: 'kind[]', value: 'reports',
                                                               label_icon: 'fas fa-flag',
-                                                              checked: @selected_filter[:kind]&.include?('reports') }
+                                                              checked: selected_filter[:kind]&.include?('reports') }
     .px-4.py-1
-      = render partial: 'webui/shared/check_box', locals: { label: 'Workflow Runs', amount: @count['workflow_runs'],
+      = render partial: 'webui/shared/check_box', locals: { label: 'Workflow Runs', amount: counted_notifications['workflow_runs'],
                                                             key: 'kind[workflow_runs]', name: 'kind[]', value: 'workflow_runs',
                                                             label_icon: 'fas fa-book-open',
-                                                            checked: @selected_filter[:kind]&.include?('workflow_runs') }
-    - if Flipper.enabled?(:content_moderation, @user)
+                                                            checked: selected_filter[:kind]&.include?('workflow_runs') }
+    - if Flipper.enabled?(:content_moderation, user)
       .px-4.py-1
-        = render partial: 'webui/shared/check_box', locals: { label: 'Appealed Decisions', amount: @count['appealed_decisions'],
+        = render partial: 'webui/shared/check_box', locals: { label: 'Appealed Decisions', amount: counted_notifications['appealed_decisions'],
                                                               key: 'kind[appealed_decisions]', name: 'kind[]', value: 'appealed_decisions',
                                                               label_icon: 'fas fa-hand',
-                                                              checked: @selected_filter[:kind]&.include?('appealed_decisions') }
+                                                              checked: selected_filter[:kind]&.include?('appealed_decisions') }
 
-  - unless @projects_for_filter.empty?
+  - unless projects_for_filter.empty?
```

Comment:
This is `@projects_for_filter` again

Reply:
Not really, it is passed as a local parameter in the `render partial` call
```
= render partial: 'notification_filter', locals: { selected_filter: @selected_filter,
                                                           counted_notifications: @counted_notifications,
                                                           user: User.session,
                                                           projects_for_filter: @projects_for_filter,
                                                           groups_for_filter: @groups_for_filter }
```